File: 1304-find-n-unique-integers-sum-up-to-zero/1304-find-n-unique-integers-sum-up-to-zero.cs
public class Solution {
    public int[] SumZero(int n) {
        var res = new List<int>();
        int l = n/2;
        for(int i=-l;i<=l;i++){
            res.Add(i);
        }
        if(n%2==0){
            res.Remove(0);
        }
        return res.ToArray();
    }
}

File: 1304-find-n-unique-integers-sum-up-to-zero/1304-find-n-unique-integers-sum-up-to-zero.cpp
class Solution {
public:
    vector<int> sumZero(int n) {
        vector<int> res;
        if(n == 0) return res;
        if(n%2!=0){
            res.push_back(0);
        }
        for(int i=-n/2;i<=n/2;i++){
            if(i==0)continue;
            res.push_back(i);
        }
        return res;
    }
};